Bowling.java

```diff
@@ -87,9 +92,8 @@ public void update(Integer pinsDown) {
 		Frame currentFrame = getCurrentFrame();
 		currentFrame.update(pinsDown);
 
-		if (currentFrame.isStrike()) {
+		if (currentFrame.isStrike())
```

Comment:
ahh...that makes sense.  Though In a perfect world just the one bowl would be fine...maybe something to revisit when we have to deal with three balls in the 10th frame.